stable/coredns/templates/clusterrole.yaml

```diff
@@ -4,15 +4,15 @@ kind: ClusterRole
 metadata:
   name: {{ template "coredns.fullname" . }}
   labels:
-    heritage: {{ .Release.Service | quote }}
-    release: {{ .Release.Name | quote }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
     {{- if .Values.isClusterService }}
     k8s-app: {{ .Chart.Name | quote }}
     kubernetes.io/cluster-service: "true"
     kubernetes.io/name: "CoreDNS"
```

Comment:
Probably the `kubernetes.io/name` label becomes redundant. better to remove. I think same applies for `k8s-app`

Reply:
I did not want to remove that for now, as I didn't want to break things for folks relying on coredns as a cluster dns service.